docs/agent-reference/how-to-run-agent.md

```diff
@@ -1,5 +1,10 @@
 # Running the Device Update for IoT Hub Reference Agent
 
+## Required Dependencies
```

Comment:
> Required Dependencies [](http://example.com/codeflow?start=2&length=22)

Recommend we add steps for adding the Microsoft Repository and how to install deliveryoptimization using 
``` sudo apt-get install deliveryoptimization

like many of our documents point out to do.

Reply:
Nic and I are following up Jimmy, meanwhile added: 

Alternatively you can add the Microsoft Repository to your local APT repositories using these (instructions)[https://docs.microsoft.com/windows-server/administration/linux-package-repository-for-microsoft-software]. Then you will be able to install the necessary DO dependencies using <bash script to install DO>